ui/app/store/actions.js

```diff
@@ -379,7 +379,8 @@ export function forgetDevice (deviceName) {
 export function connectHardware (deviceName, page, hdPath) {
   log.debug(`background.connectHardware`, deviceName, page, hdPath)
   return async (dispatch) => {
-    dispatch(showLoadingIndication())
+    const capitalizedDeviceName = deviceName ? deviceName[0].toUpperCase() + deviceName.slice(1) : ''
```

Comment:
Can we use lodash's [`capitalize `](https://lodash.com/docs/#capitalize) here?

Reply:
Good point, forgot we have lodash here 😅